### PR TITLE
Readme improvements

### PR DESCRIPTION
**Pull request by squaremo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,19 +27,23 @@ Hence we are proposing this specification, along with an initial set of plugins
 - [Weave - a multi-host Docker network](https://github.com/weaveworks/weave)
 - [Project Calico - a layer 3 virtual network](https://github.com/projectcalico/calico-cni)
 
+## Contributing to CNI
+
+We welcome contributions, including [bug reports](https://github.com/appc/cni/issues), and code and documentation improvements. If you intend to contribute to code or documentation, please read [CONTRIBUTING.md](CONTRIBUTING.md). Also see the [contact section](#contact) in this README.
```
Comment (jonboulle) [inline README.md]: Nit: elsewhere we currently stick to one-sentence-per-line, so would be nice to stay consistent here. (Would also be open to moving everything to paragraph-per-line but I suggest a follow up). 

For (way too much) historical discussion on this topic, see [here](https://github.com/coreos/docs/pull/633#issuecomment-149756526)

Comment (squaremo) [inline README.md]: Ops! Emacs reflex. Will fix (the result, not the reflex).

**Comment by jonboulle** [comment]
LGTM; travis had a hiccup so I gave it a kick.

File: README.md (hunk 1 above)
Comment (squaremo) [inline README.md]: Oh -- I misread that. Not an Emacs reflex.

**Comment by jonboulle** [comment]
:+1:

**Pull request merged**